### PR TITLE
Week 6 | Assignment 3 & 4

### PR DESCRIPTION
**Pull request by anugrahpaul2000**

### CODE REVIEW

File: Week-6/Assignment-3/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        break;
+
+      case 2:
+        count[2] += 1;
+        break;
+    }
+  }
+
+  let result = [];
+  for (const el of Object.keys(count)) {
```
Comment (subhadip-pesto) [inline Week-6/Assignment-3/index.js]: here the complexity is becoming O(n)^2

File: Week-6/Assignment-3/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+function sort(array) {
+  const count = {
```
Comment (subhadip-pesto) [inline Week-6/Assignment-3/index.js]: what if its not given that array will only contains 0,1,2 s? avoid hard coding.

File: Week-6/Assignment-4/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+function stock(prices) {
+  let profit = 0;
+
+  for (let i = 0; i < prices.length; i++) {
```
Comment (subhadip-pesto) [inline Week-6/Assignment-4/index.js]: can be done in O(n)

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

**Comment by anugrahpaul2000** [comment]
@subhadip-pesto Please Review the Assignments when you find time.

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**